bears/__init__.py

```diff
@@ -20,3 +20,25 @@ def assert_supported_version():  # pragma: no cover
     if sys.version_info < (3, 4):
         print('coala supports only python 3.4 or later.')
         exit(4)
+
+
+def check_coala_version():
+    """
+    Check the installed coala and coala-bears version when the coalib loads
+    coala-bears module.
+    """
+    try:
+        import coalib
+        if coalib.__version__ != __version__:
```

Comment:
Yes, that is much better logic.

The patch level of each package runs independently, as those patches should be bugs only within that package, and released very quickly, and thus lots of bugs in a minor release of coalib could mean its patch level increments more quickly than the bears patch level.